### PR TITLE
feat(enumeration): add loading states

### PR DESCRIPTION
**Pull request by talend-sromain**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our
  [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What is the current behavior?** (You can also link to an open issue here)

The enumeration work only in local mode

**What is the new behavior?**

The enumeration could be pluged to handler & show loader when actions are performed

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration
path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://356.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://356.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://356.talend.surge.sh)

File: output/forms.eslint.txt [hunk 1]
```diff
@@ -130,7 +130,13 @@ The react/require-extension rule is deprecated. Please use the import/extensions
   151:5   error  Value must be omitted for boolean attributes  react/jsx-boolean-value
   172:5   error  Value must be omitted for boolean attributes  react/jsx-boolean-value
 
-✖ 112 problems (112 errors, 0 warnings)
+/home/travis/build/Talend/ui/packages/forms/src/widgets/EnumerationWidget/EnumerationWidget.js
+  517:3  error  Expected indentation of 3 tabs but found 2  indent
+  521:3  error  Expected indentation of 3 tabs but found 2  indent
+  522:3  error  Expected indentation of 3 tabs but found 2  indent
+  523:2  error  Expected indentation of 2 tabs but found 1  indent
+
```
Comment (jmfrancois) [inline output/forms.eslint.txt]: please fix that

Comment (talend-sromain) [inline output/forms.eslint.txt]: it's fixed on my last commit ( ci in progress )

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://356.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://356.talend.surge.sh)

**Comment by jmfrancois** [comment]
:octocat: [Demo is available here](http://356.talend.surge.sh)

**Comment by jmfrancois** [comment]
Advice: please use shallow rendering (you have a snapshot of 6000 lines. this is useless.
And you have not unit tests on event handler.

**Pull request merged**